html/elements/optgroup.json

```diff
@@ -21,13 +21,13 @@
               "version_added": true
             },
             "firefox": {
-              "version_added": true
+              "version_added": "2"
             },
             "firefox_android": {
               "version_added": true
             },
             "ie": {
-              "version_added": false
+              "version_added": "6"
```

Comment:
What makes you say it is supported since IE 6 and not earlier? "6" and  not true here?